src/main/java/at/hannibal2/skyhanni/features/misc/customscoreboard/Events.kt

```diff
@@ -0,0 +1,531 @@
+package at.hannibal2.skyhanni.features.misc.customscoreboard
+
+import at.hannibal2.skyhanni.SkyHanniMod
```

Comment:
Please dont repeat the hard to read formatting from discord rpc here. move each logic block into its own function and not part of the enum constructors